addons/htpasswd-identity-provider/htpasswd-identity-provider.addon

```diff
@@ -1,30 +1,44 @@
 # Name: htpasswd-identity-provider
 # Description: Configures minishift to use HTPasswdIdentityProvider
 # Url: https://docs.okd.io/3.9/install_config/configuring_authentication.html#HTPasswdPasswordIdentityProvider
-# Var-Defaults: USERNAME=developer,USER_PASSWORD=developer,MINISHIFT_DATA_HOME=/var/lib/minishift,CONFIG_LOCATION=openshift.local.config/master,MASTER_CONFIG_FILE=master-config.yaml,ORIGIN_DATA_HOME=/var/lib/origin
-# Required-Vars: USER_PASSWORD,USERNAME,CONFIG_LOCATION
+# Var-Defaults: USERNAME=developer,USER_PASSWORD=developer,MINISHIFT_DATA_HOME=/var/lib/minishift,CONFIG_LOCATION_OPENSHIFT_API=base/openshift-apiserver,CONFIG_LOCATION_KUBERNETES_API=base/kube-apiserver,MASTER_CONFIG_FILE=master-config.yaml
+# Required-Vars: USER_PASSWORD,USERNAME,CONFIG_LOCATION_OPENSHIFT_API
```

Comment:
@praveenkumar Can we add version constraint here
```
# OpenShift-Version: >=3.10.10                                                            

```

Reply:
As it will not work for 3.9 or below versions.

Reply:
Yes, that we should do :+1: